backend/scripts/main.py

```diff
@@ -77,14 +78,19 @@ def GetItem():
     barcode = request.args.get('code')
     return db.GetItemFrontend(barcode)
 
+@app.get('/itemById')
+def GetItemById():
+    id = request.args.get('id')
+    return db.GetItemByID(id).toJSON() 
+
 @app.route('/make_sale', methods=['POST'])
 def MakeSale():
     content = request.json
     Items = []
     #Convert Basket from Frontend into compatible format for Printing
     for item in content:
         newItem = db.GetItemByID(item["id"])
-        newItem.SetCount(item['cnt'])
+        newItem.count = item['cnt']
```

Comment:
why are cnt and count refering to the same value (cnt == stock;   count == basket)?
Edit count where cnt is set at the moment.